### ISSUE TITLE
:bug: Inconsistent kitgun types

### ISSUE THREAD

**Issue by AyAyEm** [WFCD/warframe-items#313]
**Is your enhancement request related to a problem? Please describe.**
The value of fortuna kitguns, type key, are `Kitgun Component` and deimos kitguns, type key, are `Pistol`.

**Describe the upgrade you'd like**
Set the type key of all kitguns to `Pistol` as it makes more sense in my head.

**Additional context**
```json
[
   {
      "name":"Catchmoon",
      "type":"Kitgun Component"
   },
   {
      "name":"Gaze",
      "type":"Kitgun Component"
   },
   {
      "name":"Rattleguts",
      "type":"Kitgun Component"
   },
   {
      "name":"Tombfinger",
      "type":"Kitgun Component"
   },
   {
      "name":"Sporelacer",
      "type":"Pistol"
   },
   {
      "name":"Vermisplicer",
      "type":"Pistol"
   }
]
```

**Comment by TobiTenno** [comment]
the deimos ones should be kitgun components, because kitguns are (by definition) modular, and both pistol and not. not sure where the json you're showing is from... but that's probably been in place for a while and isn't related to #312 unless DE changed their data.

**Comment by AyAyEm** [comment]
Well the data is from the master branch and i've checked now the api: https://api.warframestat.us/items/Sporelacer?only=type
I think it's related to #312 because it changes their types i created this issue just because i was checking it.

**Comment by TobiTenno** [comment]
It changed types due to a really old catch-all from changing `/Shortguns` to 'Pistol' instead of being 'Misc'.

**Comment by TobiTenno** [comment]
If it's an issue on the pr, comment on the pr. if there's a type that you want changed, you can either put an issue for that type, or something else, i'm just confused why the initial problem (which is already in master) is being correlated to 312... which hasn't been built and deployed as of yet

**Comment by AyAyEm** [comment]
Because their types change and fixes this issue having all of the kitguns with a single type as i tried to describe in my first comment in 312

**Comment by TobiTenno** [comment]
I've addressed it in #312
